<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>advancedSeetings</class>
 <widget class="QWidget" name="advancedSeetings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Advanced settings :</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>When I sign out of Wengo :</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="forwardAllCallsToCellPhoneCheckBox" >
           <property name="text" >
            <string>Forward all my calls toward my cell phone</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Chat :</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="chatPopUpCheckBox" >
           <property name="text" >
            <string>Pop up the chat window on top of other applications when someone starts chat with me</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Calls :</string>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="callAutomaticallyAnswerCheckBox" >
             <property name="text" >
              <string>Automatically answer incoming calls after</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="callAnswerDelaySpinBox" />
           </item>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>seconds</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>callAutomaticallyAnswerCheckBox</tabstop>
  <tabstop>callAnswerDelaySpinBox</tabstop>
  <tabstop>chatPopUpCheckBox</tabstop>
  <tabstop>forwardAllCallsToCellPhoneCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="wengoconfigdata.qrc" />
 </resources>
 <connections/>
</ui>
